File: UETshare/src/main/java/com/example/uetshare/response/dto/ExamDocumentDto.java
package com.example.uetshare.response.dto;

import com.example.uetshare.entity.ExamDocumentType;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Calendar;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class ExamDocumentDto {

    private Long id;
    private String link;
    private String file_name;
    private String name;
    private ExamDocumentType type;
    private Long account_id;
    private Long subject_id;
    private SubjectDto subjectDto;
    private Calendar time;
}

File: app/src/main/java/ie/app/uetstudents/ui/Entity/subject/DataSubject/ExamDocumentDto.kt
package ie.app.uetstudents.ui.Entity.subject.DataSubject

import java.io.Serializable

data class ExamDocumentDto(
    val account_id: Int,
    val id: Int,
    val link: String,
    val name: String,
    val subject_id: Int,
    val type: String
) : Serializable